doc/src/build/move.md

```diff
@@ -386,19 +386,17 @@ to facilitate [package publishing](wallet.md#publish-packages).
 
 ```
 [package]
-name = "MyFirstPackage"
+name = "Sui"
```

Comment:
I think here the contents should be something like
```
[package]
name = "MyFirstPackage"
version = "0.0.1"

[dependencies]
Sui = { local = "../../crates/sui-framework" }

[addresses]
MyFirstPackage = "0x0"
```
according to what's in the actual Move.toml for the tutorial package (https://github.com/MystenLabs/sui/blob/main/sui_programmability/tutorial/Move.toml)?

Reply:
> I think here the contents should be something like
> 
> ```
> [package]
> name = "MyFirstPackage"
> version = "0.0.1"
> 
> [dependencies]
> Sui = { local = "../../crates/sui-framework" }
> 
> [addresses]
> MyFirstPackage = "0x0"
> ```
> 
> according to what's in the actual Move.toml for the tutorial package (https://github.com/MystenLabs/sui/blob/main/sui_programmability/tutorial/Move.toml)?

Thanks so much, Emma! This is fixed in a new commit.